apps/ocsp.c

```diff
@@ -1436,9 +1436,11 @@ static int do_responder(OCSP_REQUEST **preq, BIO **pcbio, BIO *acbio,
         *q = '\0';
```

Comment:
I am a bit confused about the syntax parsing above.

RFC 2616 says this:

5.1 Request-Line

   The Request-Line begins with a method token, followed by the
   Request-URI and the protocol version, and ending with CRLF. The
   elements are separated by SP characters. No CR or LF is allowed
   except in the final CRLF sequence.

        Request-Line   = Method SP Request-URI SP HTTP-Version CRLF


So there sould only be one ascii space char, but the first loop skips many of them, and should be
completely unnecessary.
And what about HTTP/2 ?